Assets/UnZENity-Core/Scripts/Npc/Actions/AnimationActions/Output.cs

```diff
@@ -46,7 +51,24 @@ public override void Start()
                 Props.NpcSound.PlayOneShot(audioClip);
 
                 // FIXME - Show subtitles above NPC
+                PrintDialog();
+            }
+        }
+
+        private void PrintDialog()
+        {
+            // FIXME - Show subtitles somewhere next to Hero (== ourself/main camera)
+            var currentMessage = GameData.Dialogs.CutsceneLibrary.Blocks.Find(x => x.Name == OutputName).Message;
+            NpcInstance foo = (NpcInstance)GameData.GothicVm.GlobalHero;
+            if(_isHeroSpeaking){
+                Debug.Log($"PrintDialog: {{ Hero: {currentMessage.Text}}}");
```

Comment:
I try to reduce log statements to a minimum in normal game play. My normal mode is:
1. During develepment I add Debug.Log() statements
2. Once I create the PR, I remove them again
3. If there are bugs in the future, I check via Debugger (if any)
I think you can safely do this as well. (Especially as the actual text is printed inside the Adapter the next call)